src/silo/preprocessing/preprocessing.test.cpp

```diff
@@ -960,7 +960,75 @@ const Scenario<Error> SEQUENCE_ILLEGAL_SYMBOL = {
    }
 };
 
-INSTANTIATE_TEST_SUITE_P(PreprocessorTest, InvalidPreprocessorTestFixture, ::testing::Values(DUPLICATE_PRIMARY_KEY, MISSING_NUCLEOTIDE_SEQUENCE_INPUT, TYPE_ERROR, SEQUENCE_ILLEGAL_SYMBOL), printTestName<Error>);
+const Scenario<Error> NDJSON_FILE_WITH_SOME_MISSING_KEYS = {
+   .test_name = "NDJSON_FILE_WITH_SOME_MISSING_KEYS",
+   .input_data =
+      []() {
+         std::vector<NdjsonInputLine> result;
+         result.push_back(
+            {.metadata = {{"accessionVersion", "1.1"}, {"country", "Switzerland"}},
+             .alignedNucleotideSequences = {{"main", "NNACTGNN"}, {"secondSegment", nullptr}},
+             .unalignedNucleotideSequences =
+                {{"main", "ATTAAAGGTTTATACCTTCCCAGGTAACAAACCAACCAACTTTCGATCT"},
+                 {"secondSegment", nullptr}},
+             .alignedAminoAcidSequences{
+                {"someLongGene", "ACDEFGHIKLMNPQRSTVWYBZX-*"}, {"someShortGene", "MADS"}
+             },
+             .nucleotideInsertions = {{"main", {"123:ACTG"}}, {"secondSegment", {}}},
+             .aminoAcidInsertions =
+                {{"someLongGene", {"123:RNRNRN"}}, {"someShortGene", {"123:RN"}}}}
+         );
+         result.push_back(
+            {.metadata = {{"accessionVersion", "1.3"}},
+             .alignedNucleotideSequences = {{"main", nullptr}, {"secondSegment", nullptr}},
+             .unalignedNucleotideSequences = {{"main", nullptr}, {"secondSegment", nullptr}},
+             .alignedAminoAcidSequences{{"someLongGene", nullptr}, {"someShortGene", nullptr}},
+             .nucleotideInsertions = {{"main", {}}, {"secondSegment", {}}},
+             .aminoAcidInsertions = {{"someLongGene", {}}, {"someShortGene", {}}}}
+         );
+         return result;
+      },
+   .database_config =
+      R"(
+schema:
+  instanceName: "Test"
+  metadata:
+    - name: "accessionVersion"
+      type: "string"
+    - name: "country"
+      type: "string"
+  primaryKey: "accessionVersion"
+)",
+   .reference_genomes = R"(
+{
+  "nucleotideSequences": [
+    {
+      "name": "main",
+      "sequence": "ATTAAAGGTTTATACCTTCCCAGGTAACAAACCAACCAACTTTCGATCT"
+    },
+    {
+      "name": "secondSegment",
+      "sequence": "AAAAAAAAAAAAAAAA"
+    }
+  ],
+  "genes": [
+    {
+      "name": "someLongGene",
+      "sequence": "AAAAAAAAAAAAAAAAAAAAAAAAA"
+    },
+    {
+      "name": "someShortGene",
+      "sequence": "MADS"
+    }
+  ]
+})",
+   .assertion{
+      .error_message =
+         R"(Illegal character 'E' at position 2 contained in sequence with index 0 in the current buffer.)"
```

Comment:
Where does that message come from? I wouldn't really know where to look at if I got this from SILO.